x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/Analyzer.java

```diff
@@ -2142,6 +2212,130 @@ private static Expression typeSpecificConvert(ConvertFunction convert, Source so
         }
     }
 
+    private static class ResolveUnmapped extends ParameterizedAnalyzerRule<LogicalPlan, AnalyzerContext> {
```

Comment:
That class would probably profit from a javadoc.

Reply:
Added.